Source/Core/AxisAlignedBoundingBox.js

```diff
@@ -191,6 +193,18 @@ define([
         return Intersect.INTERSECTING;
     };
 
+    /**
+     * @private
+     */
+    AxisAlignedBoundingBox.closestLocationIn = function(position, boundingObject) {
+        if (defined(boundingObject)) {
+            position.x = CesiumMath.clamp(position.x, boundingObject.minimum.x, boundingObject.maximum.x);
```

Comment:
I don't think this is right.  `position` is in world space, and `boundingObject.minimum`/`boundingObject.maximum` are relative to `boundingObject.center`